### PR TITLE
ENH make the status live!

### LINKED ISSUE #65: either remove status indicator list at bottom or make them live somehow
Right now we keep a bunch of status indicators on the page that are always green. This is confusing and I think sends the wrong message. This is especially true for things like the linter bot which is pretty flaky these days.

### PR DESCRIPTION
**Pull request by beckermr**
This PR closes #65. 

I could get 

 - [x] circle
 - [x] travis
 - [x] appveyor
 - [x] staged-recipes
 - [x] github
 - [x] docs

I could not get

- [ ] anaconda.org
- [ ] the linter bot
- [ ] azure
- [ ] drone

### CODE REVIEW

**Comment by beckermr** [comment]
here are some screen shots:



Screen Shot 2020-01-30 at 11 52 53 PM

**Comment by beckermr** [comment]
@conda-forge/core @conda-forge/bot This one is ready to go I think. I am happy to pass on the snarky title, but it seemed like fun. ;P

**Comment by CJ-Wright** [comment]
Personally I would put the cloud services lower, those are usually a given (hopefully!)

**Comment by beckermr** [comment]
Will do!

**Comment by isuruf** [comment]
I agree. I think the repos and bots section should be first, then cloud, then migrations

**Comment by beckermr** [comment]
Alright @CJ-Wright @isuruf the current order is

1. repos and bots
2. CDN
3. cloud
4. migrations
5. usage

Does this look good?

**Comment by beckermr** [comment]
Screen Shot 2020-01-31 at 8 27 20 AM

**Comment by CJ-Wright** [comment]
Personally I mostly use the status page to look at the current migration status, but I might not be the ideal user to design for.

**Comment by beckermr** [comment]
I'd say "design" is a pretty hefty word for what I am doing here. ;p

In any case, I don't really care either way, so whatever you and @isuruf want to do is fine.

**Comment by CJ-Wright** [comment]
@conda-forge/core thoughts?

**Comment by isuruf** [comment]
Usually `migrations` needs some scrolling because there are many. Even if the statuses are moved to the top, migrations will still be there in the screen right?

**Comment by beckermr** [comment]
Not quite @isuruf here is what I have

status

File: template.html [hunk 1]
```diff
@@ -38,7 +38,7 @@ <h1 class="title">{{ config.title }}</h1>
 
         <!-- {% if not panels %}
         <div class="panel operational" data-l10n-id="systems-operational">
-          All Systems Operational
+          All Systems Mostly Operational
```
Comment (CJ-Wright) [inline template.html]: I feel like this is akin to "mostly harmless"

Comment (beckermr) [reply template.html]: Happy to remove! It is just weird because we don't update that.

Comment (CJ-Wright) [reply template.html]: I didn't mean remove, I just thought the parallels with hitchhikers guide were funny

Comment (beckermr) [reply template.html]: ahhhh putting it back!

**Comment by beckermr** [comment]
OK @isuruf @CJ-Wright I reworked the page to a compromise that gets the migrations near the top. I moved the CDN status to the "repos and bots" section since it made more sense there and saved space. 

status

**Comment by beckermr** [comment]
Any other comments here @conda-forge/core?

**Pull request merged**